lib/active_admin/middleware/error_handler.rb

```diff
@@ -0,0 +1,29 @@
+module ActiveAdmin
+  module Middleware
+    class ErrorHandler
+
+      def initialize(app)
+        @app = app
+      end
+
+      def call(env)
+        @app.call(env)
+      rescue Exception => e
+        if ActiveAdmin.application.default_namespace.to_s == env['PATH_INFO'].split('/').second
```

Comment:
This code only works for the default namespace. ActiveAdmin can have many. And maybe it don't work with a root namespace.